### PR TITLE
New version: ArduinoSA.IDE.rc version 2.0.0-rc3

### PR DESCRIPTION
**Pull request by galexite**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38860)

### CODE REVIEW

File: manifests/a/ArduinoSA/IDE/rc/2.0.0-rc3/ArduinoSA.IDE.rc.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+PackageIdentifier: ArduinoSA.IDE.rc
+PackageVersion: 2.0.0-rc3
+PackageLocale: en-US
+Publisher: Arduino SA
+PublisherUrl: https://www.arduino.cc/
+PublisherSupportUrl: https://support.arduino.cc
+# PrivacyUrl: 
+Author: Arduino SA
+PackageName: Arduino IDE
```
Comment (ItzLevvie) [inline manifests/a/ArduinoSA/IDE/rc/2.0.0-rc3/ArduinoSA.IDE.rc.locale.en-US.yaml:12]:
```suggestion
PackageName: Arduino IDE RC
```

Comment (ItzLevvie) [reply manifests/a/ArduinoSA/IDE/rc/2.0.0-rc3/ArduinoSA.IDE.rc.locale.en-US.yaml:12]: to make sure it doesn't conflict with the stable version.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/ArduinoSA/IDE/rc/2.0.0-rc3/ArduinoSA.IDE.rc.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+PackageUrl: https://www.arduino.cc/en/software
+License: AGPLv3
+LicenseUrl: https://github.com/arduino/arduino-ide/blob/main/LICENSE.txt
+Copyright: Copyright (c) 2021 Arduino SA
+# CopyrightUrl: 
+ShortDescription: Arduino IDE
+Description: An integrated development environment for Arduino microcontroller boards and compatibles
+Moniker: arduino-ide
```
Comment (ItzLevvie) [inline manifests/a/ArduinoSA/IDE/rc/2.0.0-rc3/ArduinoSA.IDE.rc.locale.en-US.yaml:20]:
```suggestion
Moniker: arduino-ide-rc
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by galexite** [comment]
Corrected commit title, added `-rc` to moniker, `RC` to package name.

**Review by ItzLevvie — APPROVED** [review]

**Comment by galexite** [comment]
Just added `(RC version)\ to the description also.

**Comment by galexite** [comment]
Thank you very much for your quick review @ItzLevvie!

**Pull request merged**